endoreg_db/urls.py

```
@@ -78,12 +78,51 @@
+)
+
+# Add sensitive meta views import
+from .views.sensitive_meta_views import (
+    SensitiveMetaDetailView,
+    SensitiveMetaVerificationView,
+    SensitiveMetaListView,
+    AvailableFilesListView
+)
+
+# Add missing anonymization overview imports
+from .views.anonymization_overview import (
```

Pylint (3.3.7)

[error] Cannot import 'views.anonymization_overview' due to ''(' was never closed (endoreg_db.views.anonymization_overview, line 53)'

(E0001)